hub/core/chunk_engine.py

```diff
@@ -1539,25 +1562,27 @@ def get_single_sample(
 
         return sample
 
-    def _numpy(
+    def __numpy(
```

Comment:
Why did we change the naming of the method?

Reply:
We added a private `_numpy` with arguments that shouldnt be exposed to the user. Since that method is at a higher level than this method, we rename this to `__numpy`